packages/assets/scss/03-organisms/_rich-text.scss

```
@@ -73,6 +73,12 @@ $heading-indent: $gutter;
     }
   }
 
+  // clear float for all headings after left of right floating figures
+
+  h1, h2, h3, h4, h5, h6 {
+    clear: both;
```

```suggestion
    clear: left;
```

considering the visual regression, I think we should just clear the left float and keep the right as is